src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -501,5 +502,43 @@ protected override ITextBuffer GetBufferContainingCaret(IWpfTextView view)
         {
             return view.GetBufferContainingCaret();
         }
+
+        public List<string> GetF1Keyword()
```

Comment:
`ImmutableArray` is not marked `Serializable` and so cannot be returned from the inproc part.